src/localization/locale/en.ts

```diff
@@ -93,6 +93,8 @@ export const en = {
     artifactSearch: 'Search artifacts',
     latest: 'Latest',
     item: 'Item',
+    noData: 'No data',
+    invalidData: (docsUrl: string) => `Invalid data, see [documentation](${docsUrl}) for more information`,
```

Comment:
Nit: 
```suggestion
    invalidArtifactData: (docsUrl: string) => `Invalid data, see [documentation](${docsUrl}) for more information`,
```


I missed where this is used. But wouldn't the docs url come from localization as well? I'd almost expect this to accept the type or something about the artifact and internally determine the docs url.

Reply:
So it's only used in one place right now (empty state) but once docs are more solid I'll add other links so we can link directly to intended types if someone's data doesn't look quite right. I don't feel strongly about a ref vs a string, string just seemed clearer for the moment since I only have one ref